module/concurrent_reconfigure_test.go

```diff
@@ -47,6 +48,9 @@ const (
 )
 
 func TestConcurrentReconfiguration(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("todo: get this working on win")
```

Comment:
Without seeing what the test failures are, concurrent reconfiguration is pretty ubiquitous for robots.

Are we confident the test failure is not indicative of a real problem? We have a knob to turn off concurrent reconfiguration that we could instead use more liberally when running on windows.

Reply:
Judging by the other tests being skipped -- I'm guessing these are issues communicating with modules? Is the guess that some of our helper code for bringing up modules in tests lacking windows care?

Reply:
good callouts -- most of the failures were in the subprocess compiler (it was failing on cgo dependencies that aren't present on windows). that's not to say these *would* pass once that's fixed

now that we have a baseline of passing tests, I'm going to add back the ones that failed

I'll prioritize concurrent reconfiguration